python/librmm/pyproject.toml

```diff
@@ -15,8 +15,8 @@
 [build-system]
 build-backend = "rapids_build_backend.build"
 requires = [
+    "git+https://github.com/scikit-build/scikit-build-core.git@henryiii/fix/minversionlogic",
```

Comment:
```suggestion
    "scikit-build-core @ git+https://github.com/scikit-build/scikit-build-core.git@henryiii/fix/minversionlogic",
```

Reply:
blegh thank you. We programmatically replace this with the contents from file `dependencies.yaml` at the root of the repo, so I just updated this there and pushed https://github.com/rapidsai/rmm/pull/1638/commits/f6c2b5c4f971eb2a0082409959111cbdbc324aab.